ui/src/views/Quickstart/index.jsx

```diff
@@ -192,6 +191,28 @@ const cmdDirectory = (type, org = '<YOUR_ORG>', repo = '<YOUR_REPO>') =>
     marginLeft: theme.spacing.unit,
     marginTop: theme.spacing.double,
   },
+  taskShouldRunFlex: {
+    display: 'flex',
+    width: '100%',
+    justifyContent: 'space-between',
```

Comment:
This didn't put space between the two form groups. I don't know why. Where am I going wrong here?

Instead, I added margin-right to the form group which did the trick.

Reply:
I think we can just remove the `justifyContent` style.